### DISCUSSION TITLE
Error: File /vercel/path0/api/index.js does not exist.

### DISCUSSION THREAD

**Discussion by dan-cooke** [vercel/community#1613] [Help] [answered]
### Summary

Error when deploying a brand new remix app via the Github integration.

Works when using `vercel` cli directly.

**Comment by TooTallNate** [comment, marked as answer]
This is being tracked here: https://github.com/vercel/vercel/issues/9524

**Reply by dan-cooke** [reply]
Thank you, no idea how this doesn’t come up on google when I search it exactly. SEO on vercel docs just overrides everything
